InnoClinic/Auth.API/Controllers/AuthorizationController.cs

```diff
@@ -38,7 +40,7 @@ public async Task<IActionResult> SignIn(SignInRequest request)
                     Response.Cookies.Append("refresh", response.refreshToken);
                     return Ok(_mapper.Map<AuthorizationResponse>(response));
                 },
-                errors => Problem(errors[0].ToString()));
```

Comment:
cannot comment there but his comment refers to L:30 - why sign in is used as GET method?

Reply:
same for L:46 and sign up method